src/AddressBookUI/DisplayedPropertiesCollection.cs

```diff
@@ -117,7 +119,7 @@ IEnumerator IEnumerable.GetEnumerator ()
 
 		public IEnumerator<ABPersonProperty> GetEnumerator ()
 		{
-			var values = g ();
+			var values = g ()!;
```

Comment:
I think you can do better than using a ! since we are doing an IEnumerator, can we check if g is null and then throw the appropriate exception, for example:

```csharp
if (g is null) {
    yield break;
}
```